src/library_gl.js

```diff
@@ -720,18 +760,19 @@ var LibraryGL = {
       var context = {
         handle: handle,
         attributes: webGLContextAttributes,
-        version: webGLContextAttributes['majorVersion'],
+        majorVersion: webGLContextAttributes['majorVersion'],
+        minorVersion: webGLContextAttributes['minorVersion'],
```

Comment:
The three above code blocks also would regress code size for WebGL1 and WebGL 2. Instead of `majorVersion` and `minorVersion`, I think it'd be better to do integers `10`, `20` and `21`, the generated code size will be smaller that way.